includes/Classifai/Admin/Settings.php

```diff
@@ -28,6 +28,7 @@ public function init() {
 		add_action( 'admin_menu', [ $this, 'register_settings_page' ] );
 		add_action( 'admin_enqueue_scripts', [ $this, 'admin_enqueue_scripts' ] );
 		add_action( 'rest_api_init', [ $this, 'register_routes' ] );
+		add_action( 'after_plugin_row', [ $this, 'update_notice' ] );
```

Comment:
There's a dynamic version of this hook that uses the plugin file, allowing us to simplify here a bit. I'd suggest changing this to:
```suggestion
		add_action( 'after_plugin_row_' . CLASSIFAI_PLUGIN_BASENAME, [ $this, 'update_notice' ] );
```